core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -814,6 +815,47 @@ protected function createFooterScripts($objLayout)
 			}
 		}
 
+		// Add search index meta data
+		if ($objPage !== null)
+		{
+			$noSearch = (bool) $objPage->noSearch;
+
+			// BC, do not use $GLOBALS['TL_NOINDEX_KEYS'] anymore but make sure your page type delivers the correct
+			// <meta name="robots" content="noindex"> tag.
```

Comment:
Should we check if `TL_NOINDEX_KEYS` was modified and trigger a deprecation warning here? Like:

```php
if ($GLOBALS['TL_NOINDEX_KEYS'] && 10 !== \count($GLOBALS['TL_NOINDEX_KEYS'])) {
	@trigger_error(…);
}
```

Reply:
I don't like this. If something is deprecated, the core shouldn't make use of it anymore.